### DISCUSSION TITLE
[Projects Beta] Don't hide draft issue with any filter & add all fields to draft convert window

### DISCUSSION THREAD

**Discussion by mobile-sergey** [community/community#8129] [Projects and Issues] [answered]
We use [Project Beta] tables with filters by assigness, status and some other filters.
But when we create draft - it hiding. And we forced to use Projects in repositories instead Projects Beta.
And when draft convert to issue please add all fields from table to this window.

**Comment by azenMatt** [comment, marked as answer]
Hi @mobilase - we just added a fix here that will add values (where possible) to any items added under a filter - https://github.blog/changelog/2021-12-02-the-new-github-issues-12-02-update/ 

This should help out. 

Assignees for drafts are coming soon :)

**Reply by mobile-sergey** [reply]
@azenMatt, unfortunatelly this fix not working for me:
https://user-images.githubusercontent.com/1146694/144616318-464c739b-cd2b-4f16-8ffe-ca8c9541411e.mov

But when I change draft to issue - some fields are filled:
https://user-images.githubusercontent.com/1146694/144616602-05a62d18-7eec-4fcd-ab19-1f1a287fc7ef.mov

**Reply by azenMatt** [reply]
This is because you can't add an assignee to a draft issue - but we are working on that at the moment. 

Right now this will only work where the system is able to apply the filters to the item in question

**Reply by mobile-sergey** [reply]
Thanks for your working. I'll wait for improvements with assignee.
